README.md

```diff
@@ -1,14 +1,16 @@
 # Cloud Code for Visual Studio Code
 
-Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code works with Google’s command-line [container tools](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, and run your applications locally or in the cloud.
+Cloud Code for VS Code brings the power and convenience of IDEs to cloud-native Kubernetes and Cloud Run application development. Cloud Code integrates with a chain of Google Cloud products(like GKE, Cloud Run, Cloud APIs, Secret Manager) and makes you feel like you are working with local code. 
+
+Cloud Code works with Google’s command-line [`container tools`](https://github.com/GoogleContainerTools) like [`skaffold`](https://skaffold.dev/), [`minikube`](https://minikube.sigs.k8s.io/docs/) and [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/) under the hood, providing local, continuous feedback on your project as you build, edit, run and deploy your applications locally or in the cloud. Deeply integrates with [`Cloud SDK`](https://cloud.google.com/sdk) to provide single unified authentication in to developing services against Google Cloud.
 
 ![E2E workflow](https://github.com/GoogleCloudPlatform/cloud-code-vscode/raw/master/images/cloud-code-quick-deploy.gif)
 
 # Key Features
 
 ## Supporting your development workflow
 
-Get run-ready samples, out-of-the-box configuration snippets, support for key Google services like Google Cloud APIs and Cloud Build, one click deployment, a tailored debugging experience, and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!
+Get run-ready cloud samples, out-of-the-box configuration snippets, support for key Google Cloud Services like Google Cloud APIs and Google Cloud Build, Buildpacks, Secrets Manager, one click deployment, a tailored debugging experience, iterative run/debug experience and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!
```

Comment:
- "sample applications" instead of "cloud samples"?
- Rearranging the different parts of this sentence (ideally we should break this into two sentences but if has to be one bullet point):

Get run-ready sample applications, out-of-the-box configuration snippets, and support for key Google Cloud services like Cloud APIs, Cloud Build, Cloud Native Buildpacks, and Secret Manager, in addition to one click deployment, tailored debugging support, an iterative run/debug experience, and much more with Cloud Code — making developing with Kubernetes and Cloud Run a whole lot easier!

Reply:
Reworded & updated to two sentences.